### PR TITLE
LIME-430: Add unique profile preferences

### LINKED ISSUE #430: Enhancement: Preferences section on Public profile page should include only unique activities
Preferences section on Public profile page should include only unique types of activities that are set in user's goals
Each list item should include icon of activity and it's title

### PR DESCRIPTION
**Pull request by valery-chumak**
https://github.com/BinaryStudioAcademy/bsa-winter-2023-2024-lime/assets/72942456/a999d1f6-03b0-4a0c-ae56-baa8348de51d

### CODE REVIEW

File: frontend/src/bundles/profile/components/personal-details/personal-details.tsx [hunk 1]
```diff
@@ -99,24 +100,21 @@ const PersonalDetails: React.FC<PersonalDetailsProperties> = ({
             </div>
             <div className="my-6 w-full overflow-auto">
                 <h2 className="text-primary">Preferences</h2>
-                <ul className="mt-2 w-full gap-2 md:grid lg:grid-cols-2">
+                <ul className="mt-2 grid w-full gap-2 sm:grid-cols-2 md:grid-cols-1 lg:grid-cols-2">
                     {goals && goals.length > ZERO_VALUE ? (
-                        goals.map((goal) => (
+                        getUniqueValues(goals, 'activityType').map((goal) => (
```
Comment (stpalamar) [inline frontend/src/bundles/profile/components/personal-details/personal-details.tsx]: maybe save 'activityType' key as constant?

Comment (valery-chumak) [reply frontend/src/bundles/profile/components/personal-details/personal-details.tsx]: Fixed)

**Review by jhladkov — DISMISSED** [review]

**Review by DavidCharif — DISMISSED** [review]

File: frontend/src/bundles/profile/components/personal-details/personal-details.tsx [hunk 2]
```diff
@@ -139,7 +143,7 @@ const PersonalDetails: React.FC<PersonalDetailsProperties> = ({
                 </div>
                 <button
                     onClick={handleSendMessage}
-                    className={`${isFollowed ? 'text-action hover:border-buttonSecondary hover:text-buttonSecondary' : 'text-lm-grey-200'}  inline-flex items-center justify-center rounded-full border sm:h-14 sm:w-14 lg:h-11 lg:w-11`}
+                    className={`${isFollowed ? 'text-action hover:border-buttonSecondary hover:text-buttonSecondary' : 'text-lm-grey-200'} inline-flex min-h-12 min-w-12 items-center justify-center rounded-full border sm:h-14 sm:w-14 lg:h-11 lg:w-11`}
```
Comment (SofiiaTrokhymchuk) [inline frontend/src/bundles/profile/components/personal-details/personal-details.tsx]: use getValidClassNames

**Review by SofiiaTrokhymchuk — CHANGES_REQUESTED** [review]

File: frontend/src/bundles/profile/components/personal-details/personal-details.tsx (hunk 2 above)
Comment (valery-chumak) [reply frontend/src/bundles/profile/components/personal-details/personal-details.tsx]: Fixed)

**Review by nikita-remeslov — APPROVED** [review]

**Pull request merged**